src/WebJobs.Script.WebHost/ContainerManagement/LegionContainerInitializationHostService.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
+{
+    public class LegionContainerInitializationHostService : LinuxContainerInitializationHostService
```

Comment:
rename class to `LegionContainerInitializationHostedService` as this is an implementation of IHostedService see similar naming convention here - https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/NullHostedService.cs